SofaKernel/modules/Sofa.Config/cmake/SofaMacrosConfigure.cmake

```diff
@@ -325,7 +325,16 @@ macro(sofa_find_package name)
     if(find_package_args)
         list(REMOVE_ITEM find_package_args "BOTH_SCOPES")
     endif()
-    find_package(${name} ${find_package_args})
+
+    if(NOT TARGET ${name})
```

Comment:
Could it be ?
```
if(NOT ${name}_FOUND)
      find_package(...)
endif()    
```

To me this is working the same as the include guard...but before actually searching for instead of loading it which probably explain the difference of performance.

Reply:
I think `if(NOT TARGET ${name})` is better because `${name}_FOUND` may not be set with old custom FindXXX.cmake files.